PopupMenu/PopupMenu.module.less

```diff
@@ -11,7 +11,11 @@
 		background-image: @agate-popupmenu-bg-image;
 
 		.title {
-			margin-bottom: @agate-popupmenu-title-margin-bottom;
+			margin: @agate-popupmenu-title-margin;
+		}
+
+		.horizontalBody {
```

Comment:
This would become `.horizontal .body {` based on the above comment, and `className={css.body}` just just remain how it was.